barretenberg/cpp/src/barretenberg/api/gate_count.hpp

```diff
@@ -15,7 +15,10 @@ namespace bb {
  * @param bytecode_path Path to the file containing the serialized circuit
  */
 template <typename Builder = UltraCircuitBuilder>
-void gate_count(const std::string& bytecode_path, bool recursive, uint32_t honk_recursion)
+void gate_count(const std::string& bytecode_path,
+                bool recursive,
+                uint32_t honk_recursion,
+                bool include_gates_per_opcode)
```

Comment:
```suggestion
                bool include_gates_per_opcode)
```
We can also use this to disable collect_gates_per_opcode, so the gates command is a bit faster

Reply:
Ah yea right, updated